gui/wxpython/location_wizard/wizard.py

```diff
@@ -2259,7 +2237,7 @@ def OnEnterPage(self, event):
             label = 'matches file %s' % self.parent.filepage.georeffile
 
         elif coordsys == 'wkt':
-            label = 'matches file %s' % self.parent.wktpage.wktfile
+            label = 'matches string %s' % self.parent.wktpage.wktstring
```

Comment:
Maybe "matches WKT string" would be better.
More importantly, the string is long, so you need to change the widget to enable scrolling, it should be similar to panelProj4string in SummaryPage.